codespell_lib/data/dictionary.txt

```diff
@@ -1185,6 +1185,9 @@ amgle->angle
 amgles->angles
 amiguous->ambiguous
 amin->main, disabled because of var names
+amination->animation, amination,
+aminations->animations, aminations,
```

Comment:
I would remove the self-suggestion in this and the one above

Reply:
https://en.wikipedia.org/wiki/Amination (sorry should of linked to this before)

Reply:
also "lamination"